meta/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+galaxy_info:
+  author: Petr Kubica, Tareq Alayan, Katerina Koukiou
+  description: Role to run engine-setup.
+  company: Red Hat, Inc.
+
+  license: Apache License 2.0
+
+  min_ansible_version: 2.3
+
+  platforms:
+  - name: EL
+    versions:
+    - 7
+  - name: Fedora
+
+  galaxy_tags: [ovirt, rhv, rhev, virtualization]
+
+dependencies: []
```

Comment:
In readme you say dependecies `oVirt.repostiories` so I am just wondering whether we should depends on it really... I think we shouldn't but I am open to suggestion why we should.

Reply:
Not really dependent, just this role expect oVirt repositories enabled on the target machine. I 'd rather remove the dependency from README.

Reply:
Is there a way how to verify if oVirt.repositories are properly installed as a dependency if you install oVirt.engine-setup from Galaxy?

Reply:
It's installed as a depency in case you specify it in `meta/main.yml` as dependency, but that's something we don't want. no ?

Reply:
I think we don't want it, target machine's repos can be preconfigured.

Reply:
I agree, let's see what Martin thinks.